commands/social/store/store.js

```diff
@@ -106,26 +69,109 @@ class Store extends Social {
         
         const filter = m => m.author.id === message.author.id;
         const response = await this.client.awaitReply(message, `Are you sure you want to remove ${name} from the shop?`, filter, undefined, null);
-        if (["y", "yes"].includes(response.toLowerCase())) {
+        if (["y", "yes"].includes(response)) {
         
           await this.client.store.delete(role.id);
           message.reply("The role is now off the store.");
         } else
         
-        if (["n","no","cancel"].includes(response.toLowerCase())) {
+        if (["n","no","cancel"].includes(response)) {
           message.reply("Action cancelled.");
         }
         break;
       }
 
       case ("view"): {
         const items = message.guild.store;
-        if (items.length === 0) return message.response(undefined, "Baka... nothing is for sale!");
-        message.channel.send(`= ${message.guild.name} General Store =\n` + items.map(item => 
-          `${message.guild.roles.get(item.id.toString()).name}${" ".repeat(40 - message.guild.roles.get(item.id.toString()).name.length)}::  ${parseInt(item.price) === 0 ? "FREE" : `₲${parseInt(item.price).toLocaleString()}`} :: ${message.member.roles.has(item.id) ? "✓" : ""}`).join("\n"), { code: "asciidoc" }
+        if (!items.length) return  message.channel.send(`= ${message.guild.name} Store =\n` + "\n" + "= Tokens = \n" + `Slots${" ".repeat(20 - "Slots".length)}:: ₲${message.settings.tokenPrice} :: ${message.member.inventory["tokens"]} \n`, { code: "asciidoc" });
+        message.channel.send(`= ${message.guild.name} Store =\n` + "\n" + "= Tokens = \n" + `Slots${" ".repeat(20 - "Slots".length)}:: ₲${message.settings.tokenPrice} :: ${message.member.inventory["tokens"]} \n \n`+ "= Roles For Sale = \n"+ items.map(item => 
+          `${message.guild.roles.get(item.id.toString()).name}${" ".repeat(20 - message.guild.roles.get(item.id.toString()).name.length)}::  ${Number(item.price) === 0 ? "FREE" : `₲${Number(item.price).toLocaleString()}`} ${message.member.roles.has(item.id) ? ":: ✓" : ""}`).join("\n"), { code: "asciidoc" }
         );
       }
     }
   }
+  async buyToken(name, message, client) {
```

Comment:
you shouldn't be passing client, you can still access it with `this.client`... you're in the class